canary/apps/react/cra/package.json

```diff
@@ -10,7 +10,7 @@
     "aws-amplify": "latest",
     "react": "latest",
     "react-dom": "latest",
-    "react-scripts": "5.0.0",
+    "react-scripts": "^5.0.1",
```

Comment:
Not sure why we had pinned this, we should be tracking the latest version of `react-scripts`